commands/service_delete.go

```diff
@@ -29,39 +39,83 @@ mesg-core service delete --all`,
 		PreRunE: c.preRunE,
 		RunE:    c.runE,
 	})
-
-	c.cmd.Flags().BoolVar(&c.all, "all", c.all, "Delete all services")
-	c.cmd.Flags().BoolVarP(&c.force, "force", "f", c.force, "Force delete all services")
+	c.cmd.Flags().BoolVarP(&c.yes, "yes", "y", c.yes, `Automatically say 'Yes' to all prompts and run non-interactively`)
+	c.cmd.Flags().BoolVar(&c.deleteAllServices, "all", c.deleteAllServices, "Delete all services")
```

Comment:
s/deleteAllServices/all at it was before, at it represents the flag not the action that is taken.

Same as you did with yes - you haven't name id, confirmAllQuestions, just simple yes